data/text.js

```diff
@@ -779,6 +785,7 @@ exports.BattleText = {
 	skydrop: {
 		prepare: "[POKEMON] took [TARGET] into the sky!",
 		end: "  [POKEMON] was freed from the Sky Drop!",
+		fail: "#grassknot",
```

Comment:
This is wrong. As SadisticMystic said in the discord, Sky Drop just gets permanently grayed out against a dynamax Pokemon, similarly to Belch pre-berry-consumption.